core/src/test/java/org/apache/calcite/jdbc/CalciteRemoteDriverTest.java

```diff
@@ -880,6 +880,40 @@ public Service create(AvaticaConnection connection) {
     assertThat(updateCount, is(1));
   }
 
+  @Test public void testInsertBatchWithPreparedStatement() throws Exception {
```

Comment:
Frankly speaking, it is hard to understand what this test verifies. Could you please update test name (or even split the test into multiple tests)

Reply:
It is similar to `org.apache.calcite.jdbc.CalciteRemoteDriverTest#testInsertBatch`.The difference is the `PreparedStatement`.